bliss/encoder/encoder.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa: WPS348
```

Comment:
WPS348 is a good check, please don't disable it

Reply:
fixed it